packages/playwright/src/index.ts

```diff
@@ -185,7 +185,14 @@ export default class AxeBuilder {
 
   private async inject(frames: Frame[]): Promise<void> {
     for (const iframe of frames) {
-      await iframe.evaluate(await this.script());
+      const race = new Promise((_, reject) => {
+        setTimeout(() => {
+          reject(new Error('Script Timeout'));
```

Comment:
Should this reject descriptive be more explicit as it relates to injecting axe into the iframe? Something like: 

```suggestion
          reject(new Error('Script Timeout - Unable to inject axe into iframe'));
```

Is there a way we can also add a test for this?